src/content/blog/open-source-licence.md

```diff
@@ -0,0 +1,145 @@
+
+**GNU Licenses**
+When it comes to copyleft software licenses, the [GPL (General Public License)](https://www.gnu.org/licenses/licenses.en.html) is one of the most popular and widely used. This open-source license grants users the freedom to distribute copies and modified versions of a program as long as they retain certain conditions, such as copyright notices, warranty disclaimers, or any other licenses that have been added to the unmodified Program.
+
+By making software available with this license type, developers are ensuring that others have access to their source code, allowing them to make improvements and adaptations that benefit the community. In addition, this concept of "copyleft" ensures that anyone can collaboratively share the same freedoms when working with free software.
+
+**Mozilla Public License**
+
+The [Mozilla Public License](https://www.mozilla.org/en-US/MPL/2.0/), or MPL, is from the [Mozilla Foundation](https://foundation.mozilla.org/en/) and is also considered a weak copyleft license. The difference with this license (in comparison with the Eclipse Public License) is that it is file-based copyleft, which means code can be combined with open-source or proprietary code.
+
+**Eclipse Public License**
+
+The [Eclipse Public License](https://www.eclipse.org/legal/epl-2.0/), is from the Eclipse Foundation and is considered a weak copyleft license. A weak copyleft license requires software users to share any changes they make to the code. This license chose to implement a weaker copyleft as a way to reduce the stricter requirements users encountered with GNU’s General Public Licenses.
+
+## How Can You Choose The Correct License For Your Project?
+
+* If you're releasing your project under an open source license, choose a license compatible with the projects you're using.
+
+* If you want to allow others to use your project for commercial purposes, choose a license that permits this.
+
+* If you want to allow others to modify and distribute your project, choose a license that permits this.
+
+* If you want to ensure that your project remains free and open source, choose a copyleft license.
+
+* If you're unsure which license to choose, the Open Source Initiative has a list of popular licenses you can compare.
+
+Choosing the correct license for your open source project is an important decision. By carefully considering your needs and goals, you can choose a license that will help you to achieve your desired outcomes.
+
+## Capgo new license
+
+How do I accomplish all that? By changing my license. 
+
+Capgo has now changed the license from the MIT to a newer licensing scheme called GNU Affero General Public License V3 (AGPLv3) or any later version. Capgo version can be [found here](https://github.com/cap-go/capgo/blob/main/LICENSE.md).
```

Comment:
Wrong link it should be from capacitor updater repo, and the licence type is not correct

Reply:
Ok